### ISSUE TITLE
how does customerCode work?

### ISSUE THREAD

**Issue by osseonews** [cloudflare/stream-react#97]
We cannot get customerCode prop to work at all. The video plays without it, which is strange. We are just copying the customer code that is used in the iframe on Cloudflare for the embed, but nothing we do works. The embed itself plays fine, so there is nothing wrong with the customer. Embed shows: customer-<customerCode>.cloudflarestream.com . We tried using customerCode value, but no luck. then we tried customer-customerCode. Still doesn't work. Then we just tried, customer-customerCode.cloudflarestream.com and still nothing. What are you supposed to use for this prop?

**Comment by rtconner** [comment]
Yeah I cannot get customerCode to work either. This 100% has to be a bug in this library. 

(I think it's missing an '/iframe' from the end of the url it generates)

**Comment by oetherington** [comment]
There's an (undocumented) workaround for this where you can manually append "/iframe" to the video id and it seems to work - it's not ideal though
